pkg/object/ibmcos.go

```diff
@@ -79,10 +81,15 @@ func (s *ibmcos) Put(key string, in io.Reader) error {
 		}
 		body = bytes.NewReader(data)
 	}
+	mimeType := mime.TypeByExtension(path.Ext(key))
+	if !strings.ContainsRune(mimeType, '/') {
```

Comment:
we can have a function there these lines:  guessMimeType

Reply:
which file should I put this function on? utils.go or object_storage.go?

Reply:
utils.go is good